src/workerd/api/tests/abortable-fetch-test.js

```diff
@@ -0,0 +1,103 @@
+import {
+  strictEqual,
+  ok,
+  throws
+} from 'node:assert';
+
+// Test for the AbortSignal and AbortController standard Web API implementations.
+// The implementation for these are in api/basics.{h|c++}
+
+export const abortcontrollerAlreadyAborted = {
```

Comment:
Totally optional:

```suggestion
export const abortControllerAlreadyAborted = {
```

just to identify word boundaries for the folks who have a spell check extension installed...